addons/purchase_stock/models/purchase.py

```diff
@@ -406,6 +427,7 @@ def _create_stock_moves(self, picking):
         for line in self.filtered(lambda l: not l.display_type):
             for val in line._prepare_stock_moves(picking):
                 values.append(val)
+            line.move_dest_ids.created_purchase_line_id = False
```

Comment:
Probably need to adapt https://github.com/odoo/odoo/blob/master/addons/purchase_stock/models/stock.py#L94 in order and add a check on purchase_line_id